.github/workflows/ci.yml

```diff
@@ -44,7 +49,7 @@ jobs:
 
   bump-tag:
     name: Tag Bump
-    if: github.event_name == 'push'
+    if: github.event_name == 'push' || (github.event_name == 'create' && github.ref_type == 'branch' && startsWith(github.ref_name, 'release-')) && github.repository != 'ACCESS-NRI/model-configs-template'
```

Comment:
I have re-learnt that `&&` has higher precedence than `||` so I think the above is evaluated as:
```
A || (B && C)
```
In the above case should it be the following?
```
(A || B) && C
```

Reply:
Or is evaluated left to right in this case?

Reply:
So do you mean change:
```
push || (create && branch && release) && ! template
```
to
```
(push || create) && branch && release && ! template
```
essentially?

Reply:
Oops I meant `(push || (create && branch && release)) && ! template`? I am assuming we never want to run the step if it's the template repository

Reply:
You are correct!! Thank you for checking my logic, I missed that completely.